src/KmlViewer/XInputHelper/XInputSceneController.cs

```diff
@@ -166,8 +166,7 @@ private void UpdateView()
             if (state.Gamepad.Buttons.HasFlag(GamepadButtonFlags.DPadRight))
                 newCamera = newCamera.RotateTo(90, newCamera.Pitch, newCamera.Roll);
 
-            if (!newCamera.IsEqual(originalCamera))
-                sceneView.SetViewpointCamera(newCamera);
```

Comment:
I believe this call was pretty important. While IsEqual was removed, we should likely just do a value comparison instead.

Reply:
I added an implementation, but with my current setup I don't think I'll be able to test it.